nncf/tensorflow/quantization/algorithm.py

```diff
@@ -84,7 +84,9 @@ def __init__(self, config: NNCFConfig, should_init: bool = True):
         for quantizer_group in QuantizerGroup:
             self._parse_group_params(self.config, quantizer_group)
 
-        self._parse_init_params()
+        if self.should_init:
+            self._parse_init_params()
+
         self._range_initializer = None
```

Comment:
Call of `_parse_init_params` in condition of `self.should_init == False`  leads do Value Error, because config doesn't not contain required extra structs. CC: @negvet